common/source/docs/common-raw-imu-logging.rst

```diff
@@ -0,0 +1,23 @@
+.. _common-raw-imu-logging:
+
+======================================
+Using Raw IMU Logging for FFT Analysis
+======================================
+
+Using the :ref:`IMU Batch Sampler<common-imu-batchsampling>` to gather data for spectral analysis of noise to setup notch filtering is the historical method and still valid. But with firmware 4.5 and later, and if using an H7 -based autopilot, a better method is available and can be combined with a new analysis and filter setup tool to simply the process by logging raw IMU data instead.
+
+The downside to this method is that it produces very large logs and unless the data gathering is restricted to a single gyro, some slower autopilots (F4-based for example) may not be able to keep up with resulting logging drop-outs). Also the data produced can only be analyzed by the `Notch Filter Review tool <https://firmware.ardupilot.org/Tools/WebTools/FilterReview/>`__ on the ArduPilot Firmware "Web Tools" page.
+
+However, using this greatly simplifies the notch filtering setup and a single test flight can usually be used to gather the data and the filters then be setup interactively with the Filter Review tool.
+
+Filter Setup Process
+====================
+
+#. Select the method that will be used for dynamically adjusting the notch(s) center frequency and do any required setup, ie if :ref:`ESC telemtry<common-esc-telemetry>` will be used make sure its setup, if Throttle based make sure your battery current sensor is calibrated and setup properly, etc.
```

Comment:
```suggestion
#. Select the method that will be used for dynamically adjusting the notch(s) center frequency and do any required setup, ie if :ref:`ESC telemtry<common-esc-telemetry>` will be used make sure its setup, if Throttle based battery voltage compensation should be setup.
```

Reply:
okay thinking of compass mot for some reason...this is throttle position based....changing..thanks